trillian/ctfe/handlers.go

```diff
@@ -146,7 +146,7 @@ type AppHandler struct {
 // ServeHTTP for an AppHandler invokes the underlying handler function but
 // does additional common error and stats processing.
 func (a AppHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
-	var status int
+	var st int
```

Comment:
I think this loses some clarity, given that the type name doesn't really indicate what this variable is. Perhaps name it "statusCode" instead?

Reply:
Done.